modules/google-analytics/classes/wp-google-analytics-universal.php

```diff
@@ -49,12 +49,6 @@ public function wp_head() {
 			return;
 		}
 
-		// At this time, we only leverage universal analytics for enhanced ecommerce. If WooCommerce is not
-		// present, don't bother emitting the tracking ID or fetching analytics.js
-		if ( ! class_exists( 'WooCommerce' ) ) {
```

Comment:
Why remove this? Although the other change you made will prevent this class from being instantiated unless WooCommerce is active, this is a nice extra layer of protection that avoids this class becoming dependent on another class for sanity.